mobile/lib/screens/dashboard/dashboard_view_legacy.txt

```diff
@@ -0,0 +1,443 @@
+// ignore_for_file
+// ignore
+
```

Comment:
In the future, could we explore just having this legacy code as actual code with a version number indicated in the file name. That way, we can get to minimise commented out code in our codebase. This applies to other apps as well. Any approach to minimise "commented out" code is welcome.

Reply:
Thanks @Baalmart. This PR adds animation to the home screen. This file is the version with out animation. It shall be deleted once the UX/UI designer approves the animation otherwise, we shall roll back to this file. The reason it has a different file extension with `// ignore` and `// ignore_for_file` is to prevent dart code metrics from scanning the file and blocking merge operations.